.gitignore

```diff
@@ -9,3 +9,4 @@ target/
 .recommenders
 .mvn/wrapper/maven-wrapper.jar
 .factorypath
+eo-maven-plugin/tests-repetition.sh
```

Comment:
@volodya-lombrozo I didn't find this script in our files. Maybe you forgot to delete it?

Reply:
@Graur It's `.gitignore` file - it's the reason why you don't see it :)

Reply:
@volodya-lombrozo I suppose you can leave this change in your local files. Why do we need to add this change to our repository?

Reply:
@Graur Ok, agree. It's not good to even mention that file. Removed. Thank you!